dateparser/parser.py

```diff
@@ -155,6 +168,10 @@ def parse(cls, datestring, settings):
             order = resolve_date_order(settings.DATE_ORDER)
         else:
             order = cls._default_order
+            if bool(EIGHT_DIGIT.match(datestring)):
```

Comment:
The `if` makes the `bool()` unnecessary.

```suggestion
            if EIGHT_DIGIT.match(datestring):
```